download_entrypoint.py

```diff
@@ -1,19 +1,32 @@
-import sys
+import os
+from pathlib import Path
+from typing import List
+
+import dotenv
+import typer
 
 from downloader import download_data
 from preprocessor import preprocess_data
 
 
-DATASETS = []  # list of needed datasets in form `yyyy-mm`
-
-
-def main():
-    paths = []  # locations of downloaded datasets
-    for month in set(DATASETS + sys.argv[1:]):  # use also datasets given by user
-        path = download_data(month)
-        path = preprocess_data(path)
-        paths.append(path)
+def main(datasets: List[str] = typer.Argument(..., help='Names of datasets in format `yyyy-mm`'),
+         env: Path = typer.Option(..., exists=True, file_okay=True, dir_okay=False, readable=True,
+                                  resolve_path=True, help='Path to file with configuration enviromental variables'
+                                  )) -> None:
+    """
+    Download datasets from lichess.org and perform needed preprocessing.
+    Files will be saved into directory passed in variable `DATASETS` in given .env file.
+    """
```

Comment:
I'm not convinced about this .env file.
You are currently entering the path to the .env file to read the next path to the folder. You could just enter the path to the folder.
However, if we were to use .env, it could also contain information about what we want to download, so that we do not have to enter it in the console.
But I think that all this information could be given in the file settings.py, because we are now making 2 files with variables